src/components/Board.js

```diff
@@ -9,12 +9,19 @@ const generateSquareComponents = (squares, onClickCallback) => {
   // squares is a 2D Array, but 
   //  you need to return a 1D array
   //  of square components
+  
+  const squareComponents = [].concat(...squares);
+    return squareComponents.map((square) => {
+    return (
+      <Square key={square.id} value={square.value} id={square.id} onClick={onClickCallback}></Square>
```

Comment:
this works fine! if we'd like to go a step further and stick with standard formatting, we can put each of these props on its own line for readability:

```suggestion
      <Square 
          key={square.id} 
          value={square.value} 
          id={square.id} 
          onClick={onClickCallback} 
      />
```